src/frontend/contexts/wishlist-context.js

```diff
@@ -0,0 +1,68 @@
+import { useState, useEffect, createContext, useContext } from "react";
+import { useAuth } from "./auth-context";
+import axios from "axios";
+
+const WishlistContext = createContext();
+
+const WishlistProvider = ({ children }) => {
+  const { auth } = useAuth();
+
+  const [wishlist, setWishlist] = useState([]);
+  console.log("wishlist", wishlist);
+
+  useEffect(() => {
+    if (auth.status) {
+      console.log("wishlist if");
```

Comment:
You forgot to remove this console I guess.

Reply:
Was testing something. Thank you!